app/helpers/orchid/application_helper.rb

```diff
@@ -75,6 +75,20 @@ def clear_search_text
     return new_params
   end
 
+  def locale
+    params["locale"] || APP_OPTS["language_default"]
+  end
+
+  def localized_partial(partial_name, prefixes)
+    localized = "#{partial_name}_#{locale}"
+    if lookup_context.template_exists?(localized, prefixes, true)
+      "#{prefixes}/#{localized}"
+    else
+      # fallback to informative partial about customization
+      "errors/missing_partial"
```

Comment:
Yay! Nice that the partial you render can give the locale in your instructions. I wonder if an `@instance_var` could be used to give the `"#{prefixes}/#{localized}"` string it was looking for too? Could save us a little time in the future? :thinking:

Reply:
Interesting idea about the instance variable, I didn't think about that.  I did toy slightly with the notion of passing the current location to the partial as a local variable, although then I didn't just because I figured most of the time the correct partial wouldn't need it.  I do like the idea of leaving ourselves hints for the future, though, so I'll look into that more.